Missing driver property of onPopoverRender in Config type

I think in 333d0df15ba1defbd915936a5c32377d7d9ee68d the type definition for the `driver` property of `onPopoverRender` in the `Config` type was forgotten. 
https://github.com/kamranahmedse/driver.js/blob/e2a780ee3a5339fba39be9013fa9d1aa42e77f2f/src/config.ts#L41

Currently, I get `error TS2339: Property 'driver' does not exist on type '{ config: Config; state: State; }'.` when trying to use the `driver` property in Version 1.3.4 even though the property is actually passed to `onPopoverRender` in https://github.com/kamranahmedse/driver.js/blob/e2a780ee3a5339fba39be9013fa9d1aa42e77f2f/src/popover.ts#L223

--- a/src/config.ts
+++ b/src/config.ts
@@ -38,7 +38,7 @@ export type Config = {
   doneBtnText?: string;
 
   // Called after the popover is rendered
-  onPopoverRender?: (popover: PopoverDOM, opts: { config: Config; state: State }) => void;
+  onPopoverRender?: (popover: PopoverDOM, opts: { config: Config; state: State, driver: Driver }) => void;
 
   // State based callbacks, called upon state changes
   onHighlightStarted?: DriverHook;

